_opensearch/remote.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Remote-backed storage
+nav_order: 19
+---
+
+# Remote-backed storage
+
+Remote-backed storage offers OpenSearch users a new way to protect against data loss by automatically creating backups of all index transactions and sending them to a supported cloud storage service. In order to expose this feature, segment replication must also be enabled. See [Segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/) for additional information.
+
+Remote-backed storage is an experimental feature. Therefore, we do not recommend the use of remote-backed storage in a production environment. For updates on the progress of remote-backed storage, or if you want leave feedback that could help improve the feature, refer to the issue on [GitHub](https://github.com/opensearch-project/OpenSearch/issues/1968).
+{: .note}
+
+## Enable the feature
+
+In order to create new indexes with remote-backed storage enabled, you must first enable these features by adding the correct properties to `run.gradle` before building OpenSearch. See the [developer guide](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md) for information about how Gradle is used to build OpenSearch.
+
+Add the following properties to `run.gradle` to enable the feature:
+
+```bash
+testClusters {
+  runTask {
+    testDistribution = 'archive'
+    if (numZones > 1) numberOfZones = numZones
+    if (numNodes > 1) numberOfNodes = numNodes
+    systemProperty 'opensearch.experimental.feature.replication_type.enabled', 'true'
+    systemProperty 'opensearch.experimental.feature.remote_store.enabled', 'true'
+  }
+}
+```
+
+Segment replication must also be enabled in order to use remote-backed storage.
+{: .note}
+
+After building OpenSearch with these properties, you must enable the feature for all nodes in your cluster. This can be done by modifying a `config/jvm.options`, or by defining `OPENSEARCH_JAVA_OPS` from the command line.
+
+### Option 1: Modify jvm.options
+
+Add the following lines to `config/jvm.options` before starting the OpenSearch process to enable the feature and its dependency:
```

Comment:
turn on

Reply:
Leaving as enabled. Style guide does not demand that enable be avoided.

Use turn on, activate, or start instead of enable to support bias-free documentation, when possible. Otherwise, use enable to describe making a feature or command available. If the UI uses active and inactive to describe these states, use activate and deactivate in the documentation.

Avoid using enable to refer to making something possible for the user. Instead, rewrite to focus on what's important from the user's point of view. For example, “With ABC, you can do XYZ” is a stronger statement than “ABC enables you to XYZ.” Additionally, using a task-based statement is usually more clear than the vague “…enables you to….”